app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -52,6 +72,14 @@
               <%= feedback_message.message %>
             <% end %>
           </p>
+          <% else %>
+            <h5 class="fw-bold">
+            Message from Offender:
+          </h5>
+          <div class="reported__message">
+                 <%= raw(feedback_message.message) %>
```

Comment:
The formatting on this file is kinda wonky. I'm not super familiar with rubocop/ruby linting but is there an option to lint the `erb` files?

Reply:
Rubocop and I believe erblint handle this. We have a precommit hook that does this.